package.json

```diff
@@ -4,6 +4,8 @@
   "description": "A to do application",
   "main": "index.js",
```

Comment:
## Nitpick

Let this point to the actual entry point we have defined. Since there is no file called `index.js` in the root, this can be a bit misleading.